packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -70,13 +71,17 @@ function aggregateTranslationMessages(options, inputFileSystem) {
   const languageMessages = {};
   supportedLocales.forEach((language) => { languageMessages[language] = {}; });
 
-  // Read translations at the baseDirectory
-  readTranslations(languageMessages, options.baseDirectory, inputFileSystem);
-  // Set translationsDirectory to root node_modules if not specified
-  const translationsDirectory = options.translationsDirectory || ['node_modules'];
-  // Search and aggregate translation messages under the custom directories
-  translationsDirectory.forEach(dir =>
-    aggregateDirectory(languageMessages, path.resolve(options.baseDirectory, dir), inputFileSystem),
+  // Read translations directly from specific paths
+  if (options.translationsDirectories) {
+    options.translationsDirectories.forEach(dir =>
+      readTranslations(languageMessages, path.resolve(options.baseDirectory, dir), inputFileSystem),
```

Comment:
Since we are providing absolute paths now, would it make since to remove the baseDirectory restriction and just have 

 ```
readTranslations(languageMessages, dir, inputFileSystem),
```

Reply:
This option expects an array of paths relative to baseDirectory, e.g.
```
[folder1/folder2, ../folder3]
```
The goal is to make the config simple instead of
```
[path.resolve(__dirname, folder1/folder/2), path.resolve(__dirname, ../folder3)]
```